invisible_cities/sierpe/fee_test.py

```diff
@@ -53,6 +53,73 @@ def test_fee_params():
     assert FE.f_LPF2 == 10 * units.MHZ
     assert FE.OFFSET == 2500   # offset adc
 
+def test_show_scipy_nasty_feature():
+    """
+    Scipy 1.1.0 changes shamelessly the default behaviour of signal.decimate
+    From the manual:
+
+    v 0.19
+
+    scipy.signal.decimate(x, q, n=None, ftype='iir', axis=-1, zero_phase=None)[source]
+    Downsample the signal after applying an anti-aliasing filter.
+
+
+    Parameters:
+    x : ndarray
+    The signal to be downsampled, as an N-dimensional array.
+    q : int
+    The downsampling factor.
+
+    n : int, optional
+    The order of the filter (1 less than the length for ‘fir’).
```

Comment:
The signal to noise ratio here is awful. Rather than copypasting the whole manual entries for the two versions, I would just describe the change: `n` used to have these defaults, now it has these, and maybe leave a link to the manual.